test/methods.jl

```diff
@@ -299,6 +299,16 @@ end
         zonal(sum, st; of=dims(st)) == 
         zonal(sum, st; of=Extents.extent(st)) == 
         sum(st)
+
```

Comment:
Thanks! My last nitpick is to put all of this in a `@testset` with a suitable name

Reply:
I've separated them into a test set called `zonal_return_missing`. Feel free to suggest a better name, naming is hard!